BiweeklyMeetings/2022-05-10.md

```diff
@@ -0,0 +1,494 @@
+
+**M. Àngels**-:
+Now I have added to the agenda, some issues. Do you want to review this or do you want to leave them for after the discussion point, which are the CIP 50 and the one you brought at the beginning?
+
+**Matthias**-:
+Yeah. Let's first talk about CIP 50 and the audit guidelines first so that we have time. And then if time is left, we can go for issues, which are a bit more minor points.
+
+**M. Àngels**-:
+Okay.
+
+**Matthias**-:
+Do we have Kevin today or not? No...
+
+**M. Àngels**-:
+We have Michael but not Kevin.
+
+### Last Check 
+
+### Review 
+
+### Discussions 
+
+#### PR242
+ [PR242 CIP-0050? | Shelleys Voltaire decentralization update](https://github.com/cardano-foundation/CIPs/pull/242)
+
+
+**Matthias**-:
+Yeah. So maybe Michael knows, actually. So there was this discussion a few weeks ago to start a study group on CIP 50 with people from IOG and this was something Kevin picked up and was sorting. I'm not sure if it has happened yet or no. It hasn't happened yet Michael, okay. So that's something we'd need to follow up with. I know Kevin did reach out to a few people and it was acting behind the scene to get it done, but yeah, I will just reach out to him. Yes, definitely. Vasil has brought some attention already. But yeah, there are people I think we can, from which we can pick their brain to just stop thinking about that who are not directly involved in Vasil. Okay. I will just reach out to Kevin and ask him if he has any update on that regarding this study group. Maybe you can bring Michael on screen, if just for a short of date, maybe you have had also some discussions on the site already or got any feedback that you want to share?
+
+**Michael**-:
+All right, here we go. There's always many discussions on the side about this particular CIP, the very important CIP in many ways. So this will be continued to be [inaudible 00:16:12] feedback of individuals including yourself.
+
+**Matthias**-:
+Yeah. Okay. So I guess this is also one of the CIPs where it would be worth maybe giving some summary of discussions every week. So I haven't gone through the whole discussion thread I admit yet, but I will try to yeah... Post summaries a bit like we did for the... Oh, what was it? The NFT mid standards. And CIP 30 actually, back in the days. So we get some yeah, regular point of synchronization where we can move the discussions a bit forwards.
```

Comment:
```suggestion
Yeah. Okay. So I guess this is also one of the CIPs where it would be worth maybe giving some summary of discussions every week. So I haven't gone through the whole discussion thread I admit yet, but I will try to yeah... Post summaries a bit like we did for the... Oh, what was it? The NFT standards. And CIP 30 actually, back in the days. So we get some yeah, regular point of synchronization where we can move the discussions a bit forwards.
```